src/dimension_slice.c

```diff
@@ -983,6 +1025,216 @@ ts_dimension_slice_insert_multi(DimensionSlice **slices, Size num_slices)
 	return n;
 }
 
+/*
+ * Insert correlated constraint dimension slices into the catalog.
+ *
+ * Create entries with MINVALUE/MAXVALUE as ranges for correlated
+ * constraints and insert these entries. This allows for all the
+ * chunks to be picked up when queries use correlated columns in
+ * WHERE clauses.
+ *
+ * Also, add these to the chunk's constraint lists so that they
+ * eventually get added into the metadate in this same transaction
+ *
+ * We will asynchronously convert the entries into proper ranges
+ * later
+ *
+ * Returns the number of slices inserted.
+ */
+int
```

Comment:
```suggestion
void
```
The caller is not using the returned value.

Reply:
This is consistent with the other `ts_dimension_slice_insert_multi` API.